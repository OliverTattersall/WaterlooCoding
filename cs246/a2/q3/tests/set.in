a 0 b 1 1 4 4
pc0
s c 0 0 y 
pc0
s t 0 0 1 1 
pc0
s s 0 0 2 2
pc0
s s 0 0 2 4
pc0
