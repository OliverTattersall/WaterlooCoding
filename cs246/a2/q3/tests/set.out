Dimensions: 5x5
Number of rectangles: 1
	[Blue (1,1) 4x4]
Dimensions: 5x5
Number of rectangles: 1
	[Yellow (1,1) 4x4]
Dimensions: 6x6
Number of rectangles: 1
	[Yellow (2,2) 4x4]
Dimensions: 10x10
Number of rectangles: 1
	[Yellow (2,2) 8x8]
Dimensions: 18x34
Number of rectangles: 1
	[Yellow (2,2) 16x32]
